requirements.txt

```diff
@@ -3,4 +3,5 @@ uritemplate>=3.0.0
 promise>=1.0.1
 marshmallow>=3.10.0
 future>=0.18.3
-urllib3<2.0
+urllib3==1.26.15
```

Comment:
Why not `urllib3<2.0` ? It literally means _install anything lower but 2.0_

Reply:
`1.26.15` is the last version before 2.0.0

Reply:
The thinking goes: It's the latest now, but you don't know if there's a security bug uncovered later and a patch released as `1.26.16`. Anyway...